source/Weapon.h

```diff
@@ -74,6 +74,7 @@ class Weapon {
 	const std::map<const Effect *, int> &TargetEffects() const;
 	const std::map<const Effect *, int> &DieEffects() const;
 	const std::vector<Submunition> &Submunitions() const;
+	int FadeOut() const;
```

Comment:
I think this should go after `RandomLifetime()`.

Reply:
ok